optimize/client/src/modules/icons/CamundaLogo.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import {SVGProps} from 'react';
+
+export default function CamundaLogo(props: SVGProps<SVGSVGElement>): JSX.Element {
+  return (
```

Comment:
Why are we converting the SVGs to react components?
Can we just import them as react component directly like it is described here?:
https://www.npmjs.com/package/vite-plugin-svgr